composefs.spec.in

```diff
@@ -22,6 +27,7 @@ Devel files for %{name}.
 
 %package        libs
 Summary:        Libraries for %{name}
+License:        (LGPL-2.0-or-later AND Apache-2.0) AND (LGPL-2.1-or-later AND (GPL-2.0-only OR BSD-2-Clause))
```

Comment:
Which part is lgpl 2.0? I was expecting only lgpl 2.1.

Isn't the correct list more like:
 LGPL-2.1-or-later AND // all our code, and bitrotate.h and hash.[ch] 
(GPL-2.0-only OR Apache2.0) AND // erofs_fs.h
(GPL-2.0-only OR BSD-2-Clause))  // xxh32 hash

Reply:
on review I can't see it either. I put the various licenses found in a reply above.